app/services/CheckoutService.scala

```diff
@@ -276,14 +278,18 @@ class CheckoutService(identityService: IdentityService,
     val ratePlan = RatePlan(renewal.plan.id.get, None, Nil)
     val amend =   Amend(subscriptionId = subscription.id.get, plansToRemove = Nil, newRatePlans = NonEmptyList(ratePlan))
 
+    val contractEffective = subscription.termEndDate
+    val customerAcceptance = nextFridayFrom(contractEffective)
```

Comment:
Just a note that when we offer 'x issues free' renewal promotions, it's this customerAcceptance that will need extending by X-from-the-promotion days.